docs/manual/en/manual.txt

```diff
@@ -0,0 +1,258 @@
+
+The sustain (S) parameter controls the volume level held after A + D has elapsed, and as long as the key remains pressed.
+However, if the "Sustain" flag is disabled, there is no sustain stage, and the envelope effectively becomes an "ADR".
+
+The release (R) parameter comes after sustain, and similarly to "A" and "D", specifies the duration of the release stage.
+
+==== Flags
+
+The "Tremolo" and "Vibrato" flags apply small modulations to the operator, affecting volume and frequency respectively.
+
+"Sustain" enables or disables the "S" stage of the envelope. Refer to the explanation on envelopes.
+
+"Key scaling" automatically shortens the duration of the envelope as the pitch of the note increases.
+
+==== Levels
+
+The output level "Lv" is the global volume of the operator.
+Note that there is no "0" setting; even at the lowest level, the operator can be heard faintly.
+
+The frequency multiplier "F*" controls the base frequency of the operator, in octaves.
+
+The key scale level "Ksl" determines the extent to which amplitude is attenuated as pitch increases.
+
+=== Other parameters
+
+The remaining parameters fall into two groups: Miscellaneous and Global.
+
+A miscellaneous parameter applies to the currently selected program in the editor. On the other hand, a global parameter is a property of the bank, and applies to all programs in that bank.
+
+.Miscellaneous parameters
+image::../resources/misc.png[200,200]
+
+"Velocity Offset" is a fixed modifier applied to the velocity of MIDI notes, and can be set in the range [0, 127]. A positive offset increases velocity, and a negative value decreases it.
+
+.Global parameters
+image::../resources/global.png[230,230]
+
+The "Deep tremolo" and "Deep vibrato" flags make those effects more pronounced, if they are active. The effect depths are fixed values determined by the design of the YMF262.
+
+"Volume moodel" specifies a function for transforming the discrete parameters associated with volume into actual gain values. In most cases, the generic model is appropriate.
+
+However, when loading banks from other software, the formula that reproduces the operation of that software should be selected. For example, if you load a bank from Doom, you should choose the DMX model. For Duke Nukem 3D, use Apogee.
+
+== About
+
+ADLplug was developed by Jean Pierre Cimalando in 2018. +
+It is based on previous work by several groups of people.
+
+The YMF262 emulators were developed by the DOSBox team, and by Alexey Khokholov (Nuke.YKT) for Nuked OPL.
+
+The MIDI driver was previously developed by Joel Yliluoma (Bisqwit). Originally, this program, ADLMIDI, was a simple MIDI file player.
+
+Vitaly Novichkov (Wohlstand) reorganized ADLMIDI into a reuseable library, adding the basic building blocks of realtime MIDI control. Jean Pierre Cimalando has contributed improvements which make it more suitable for usage in MAO under real-time constraints.
```

Comment:
What is MAO?

Reply:
MAO =! DAW
MAO = computer aided music

Reply:
Thank you for the clarification. I'm going to go with "Computer music" as in the Wikipedia page:

https://fr.wikipedia.org/wiki/Musique_assist%C3%A9e_par_ordinateur -> https://en.wikipedia.org/wiki/Computer_music

"Computer Aided Music" in English makes me think of algorithmically generated music.